What's new for .NET 9 Preview 6 (#41745)
dotnet/docs: docs/core/compatibility/sdk/8.0/dotnet-restore-audit.md

## Reason for change

> In most cases when you restore a package, you want to know whether the restored package version contains any known security vulnerabilities. This functionality was a highly requested feature. Security concerns continue to increase each year and some known security issues aren't visible enough to taking immediate action.

Reviewer: ```suggestion
Many users want to know whether the packages they restore contain any known security vulnerabilities. This functionality was a highly requested feature. Security concerns continue to increase each year and some known security issues aren't visible enough to take immediate action.
```
